test/acceptor_SUITE.erl

```diff
@@ -916,6 +917,30 @@ do_tcp_local_echo() ->
 		file:delete(SockFile)
 	end.
 
+tcp_socket_echo(_) ->
+	case erlang:function_exported(erlang, atom_to_binary, 1) of
```

Comment:
I don't know what version you are targetting from just reading the code, and I also don't know why you only target those versions. Need a comment at the very least.

Reply:
I added an explanation. Is it enough to know what is going on, or should I be more elaborate?

Reply:
Yup thanks!